s/features/notes/components/notes/xiome-notes.css.ts

```diff
@@ -3,16 +3,82 @@ import {css} from "../../../../framework/component.js"
 export default css`
 
 :host {
-	display: block;
+	display: flex;
+	flex-direction: column;
+	align-items: stretch;
+	justify-content: center;
+	max-width: 50em;
+	border: solid 0.1em;
+}
+
+xio-button {
+  --xio-button-border: none;
+}
+
+/* Note tabs new/old */
+
+.tabs {
+	display: flex;
+	justify-content: center;
+	padding-bottom: 1em;
+	margin: 1em 2em;
+	border-bottom: solid 0.1em;
 }
 
 [data-tab] {
 	font-weight: normal;
+	margin: auto;
 }
 
 [data-tab][data-active='true'] {
 	font-weight: bold;
 	color: bisque;
```

Comment:
let's remove this line about `bisque` -- better to let the underline indicate the current tab